templates/zerver/help/index.md

```diff
@@ -97,8 +97,6 @@ as an **organization**.
 * [Change the color of a stream](/help/change-the-color-of-a-stream)
 * [Message a stream by email](/help/message-a-stream-by-email)
 * Convert a group PM to a private stream (not implemented)
-* [Remove someone from a stream (admin only)](/help/remove-someone-from-a-stream)
-* [Delete a stream (admin only)](/help/delete-a-stream)
```

Comment:
Yes; they have already been included in the "Administering a Zulip organization" section.